src/creatures/players/player.hpp

```diff
@@ -2655,6 +2665,41 @@ class Player final : public Creature, public Cylinder, public Bankable {
 
 	uint16_t getPlayerVocationEnum() const;
 
+	// Store functions
+	void openStore() {
```

Comment:
It's a good practice to move the body of these functions to the .cpp file, which helps to avoid increasing the complexity of the .hpp file even further.